user_pss_adaptive_server_spec.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Swarm PSS adaptive server spec
+about: Swarm PSS adaptive server spec
+title: Swarm PSS adaptive server spec
```

Comment:
I'm not sure we need special server-side considerations for `pss`.

Reply:
Ok, so perhaps a general `user_adaptive_server_spec` would suffice?